pkg/operator/operator.go

```diff
@@ -136,3 +128,10 @@ func getGeneration(client appsclientv1.AppsV1Interface, ns, name string) int64 {
 	}
 	return deployment.Generation
 }
+
+func (c serviceCAOperator) updateStatus(operatorConfigCopy *operatorv1.ServiceCA) {
```

Comment:
nit: that the argument is a Copy does not seem relevant to the function so I would suggest just operatorConfig

Reply:
done